Templates/Specimens_Metadata_2021Q1_template.tsv

```diff
@@ -0,0 +1,18 @@
+Number of subspecimens	A count of specimens derived from this specimen (cumulative)	APOLLO_SV:00000032	count	A measurement datum that is the output of counting.
+Specimen type	Type of specimen (excised tissue block, slice, cell, etc.)	AIO:specimen type name	specimen type name	A textual entity that denotes the type of a specimen.
+Hemisphere	Hemisphere that this specimen came from (left, right, both, unknown)	PATO:0000140	position	A spatial quality inhering in a bearer by virtue of the bearer's spatial location relative to other objects in the vicinity.
+Anatomical structure	Specimens location within anatomical structures from brain atlas	PATO:0000141	structure	A morphology quality inhering in a bearer by virtue of the bearer's relative position, shape, arrangements and connectivity of an organism's various parts; the pattern underlying its form.
+Specimen disease condition	Description of specimen’s disease state (healthy vs diseased tissue)	OBI:0000278	disease stage	A part of an occurrence of a disease process which is associated with position in the normal progression of the disease
+Subject organism ID	ID for the subject organism that this specimen came from	OBI:0100026	organism	A material entity that is an individual living system, such as animal, plant, bacteria or virus, that is capable of replicating or reproducing, growth and maintenance in the right environment. An organism may be unicellular or made up, like humans, of many billions of cells divided into specialized tissues and organs.
+Organism age time measurement	Age of organism at the time of initial specimen creation	OBI:0001167	age measurement datum	A time measurement datum that is the result of measurement of age of an organism
+Organism age measurement unit	Unit for age measurement (days, months, years..)	OBI:0001167	age measurement datum	A time measurement datum that is the result of measurement of age of an organism
+Organism age measurement type	Type of age measurement (postnatal, embryonic)	OBI:0001167	age measurement datum	A time measurement datum that is the result of measurement of age of an organism
+Organism phenotypic sex	Description of organism’s sex phenotype (female, male, etc.)	PATO:0001894	phenotypic sex	An organismal quality inhering in a bearer by virtue of the bearer's physical expression of sexual characteristics.
+Organism life cycle stage	Description of organism’s life cycle state (juvenile, adult, elderly)	UBERON:0000105	life cycle stage	A spatiotemporal region encompassing some part of the life cycle of an organism.
+Organism genotype	A string describing the genotype of the organism	OBI:0001305	genotype information	a genetic characteristics information that is about the genetic material of an organism and minimally includes information about the genetic background and can in addition contain information about specific alleles, genetic modifications, etc.
```

Comment:
Might also want strain RRID or similar.

Reply:
As part of the NWB project, we spent sometime figuring how to model genotype
https://docs.google.com/document/d/12EVToclVEKXF8OP5hd6eaTYT6vj-ewhHYVM54u7h-tU/edit

Reach out if you want a walk through

Reply:
I changed the description to say the genotype should be formatted according to NWB convention.